ziti/config/options.go

```diff
@@ -0,0 +1,26 @@
+package config
```

Comment:
Should ListenOptions move here as well?

Reply:
I think so
related to this I'd like to refactor `listenManager` into a separate file at some point